lua/lean/infoview/components.lua

```diff
@@ -0,0 +1,84 @@
+---@brief [[
+--- Infoview components which can be assembled to show various information
+--- about the current Lean module or state.
+---@brief ]]
+
+---@tag lean.infoview.components
+
+local components = {}
+
+local _SEVERITY = vim.tbl_extend("error",
+  { [0] = "Other" },
```

Comment:
If you want to reuse the built-in DiagnosticSeverity map, then you can drop the Other case.  I've only included it for completeness, but it's not used by Lean afaict.

Reply:
Done.